mlflow/entities/trace_info_v3.py

```diff
@@ -28,7 +28,11 @@ class TraceInfoV3(_MlflowObject):
     assessments: list[Assessment] = field(default_factory=list)
 
     def to_dict(self) -> dict[str, Any]:
-        return MessageToDict(self.to_proto(), preserving_proto_field_name=True)
+        res = MessageToDict(self.to_proto(), preserving_proto_field_name=True)
+        if self.execution_duration is not None:
+            res.pop("execution_duration", None)
+            res["execution_duration_ms"] = self.execution_duration
+        return res
```

Comment:
We could add a separate method for inference table JSON representation, but it's confusing and error-prone if we have multiple to_dict variants.

Reply:
I agree that we should keep one `to_dict` variant. Standardizing on this one format should be simplest in the long run